### PR TITLE
completed controller actions and routes

### PR DESCRIPTION
**Pull request by unlimitedFinite**
Have added routes to 'config/routes'

Have added basic method within the BookingsController

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+class BookingsController < ApplicationController
+  before_action :set_booking, only: [:show, :destroy]
+
+  def index
```
Comment (spector-jake) [inline app/controllers/bookings_controller.rb]:
```suggestion
  def index
    @bookings = Booking.all
```

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  end
+
+  private
+
+  def all_bookings
+    @bookings.all
+  end
+
+  def set_booking
+    @booking.find(params[:id])
```
Comment (spector-jake) [inline app/controllers/bookings_controller.rb]:
```suggestion
    @booking = Booking.find(params[:id])
```

File: app/controllers/bookings_controller.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  end
+
+  def booking_params
+    params.require(:booking).permit(:start_dt, :end_dt, :price_paid)
```
Comment (spector-jake) [inline app/controllers/bookings_controller.rb]: Should we specify price_paid? The user shouldn't enter this value. Should be calculated from the number of days and pokemons.price_per_day

File: app/views/bookings/delete.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<h1>Bookings#delete</h1>
```
Comment (spector-jake) [inline app/views/bookings/delete.html.erb]: I don't think this file is needed. Should be deleted

**Comment by unlimitedFinite** [comment]
Yeah good point, I will change it

File: app/views/bookings/view.html.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<h1>Bookings#view</h1>
```
Comment (spector-jake) [inline app/views/bookings/view.html.erb]: What is this file for? View isn't one of the 7 routes

**Comment by unlimitedFinite** [comment]
accidentally generated during ‘rails g controller’ command

File: app/views/bookings/create.html.erb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+<h1>Bookings#create</h1>
```
Comment (spector-jake) [inline app/views/bookings/create.html.erb]: This file isn't needed. Should delete it

**Pull request merged**